### DISCUSSION TITLE
Password protected R Markdown

### DISCUSSION THREAD

**Discussion by adamkucharski** [WHO-Collaboratory/collaboratory-epipipeline-community#18] [Ideas]
WHO Collaboratory Hackathon Idea.

**Comment by Bisaloo** [comment]
This package provides a good solution but it hasn't been updated in a couple of years and would probably benefit from contributions (TBC with the package maintainer): https://github.com/dirkschumacher/encryptedRmd

**Reply by ntncmch** [reply]
Other potentially interesting packages:
- https://github.com/mattwarkentin/fidelius/
- https://github.com/favstats/rmdprotectr

**Reply by Bisaloo** [reply]
Another new package for this purpose: https://cran.r-project.org/package=staticryptR
